config/deploy.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+lock '3.5.0'
+
+set :application, 'st_pax'
+
+set :scm, :git
+set :repo_url, 'git@github.com:umts/st-pax.git'
+set :branch, :master
+
+set :keep_releases, 5
+
+set :deploy_to, "/srv/#{fetch :application}"
+
+set :log_level, :info
+
+set :whenever_command, %i[sudo bundle exec whenever]
```

Comment:
We should make sure that `PATH` is set correctly in the `config/schedule.rb` file.

See https://github.com/umts/jobapps/pull/300.

Reply:
I don't. What.

Reply:
Literally, just that line?

Reply:
Yes, we should add that line.